boilerplate/fullstack/remix/app/lib/superTokensHelpers.ts

```diff
@@ -92,88 +93,31 @@ export function createPreParsedRequest(request: Request): PreParsedRequest {
 }
 
 export async function getSessionDetails(
-    request: Request,
+    baseRequest: PreParsedRequest | Request,
```

Comment:
this is very strange. We have to be very clear.. request must be one of Remix request or PreParsedRequest. Not OR.